nixos/modules/services/matrix/synapse.nix

```diff
@@ -286,6 +286,7 @@ in {
             log_config = mkOption {
               type = types.path;
               default = ./synapse-log_config.yaml;
+              defaultText = lib.literalExpression "nixos/modules/services/matrix/synapse-log_config.yaml";
```

Comment:
I'm not sure what should be done with this. Maybe `toPretty` should somehow be made aware of `pkgs.path` so that it can render internal paths independently of where nixpkgs is located?

Reply:
The documentation generator already needs to deal with paths for the declaration and definition locations. The logic for this is a bit strewn about. It'd make sense to make the generic doc generation take ownership of that functionality and then reuse that logic by passing it to `toPretty`.